src/scripts.js

```diff
@@ -71,11 +71,14 @@ function onStartup() {
 
 function compilePantryData(recipesList) {
   recipesList.forEach(recipe => {
-    let missingIngredients = pantry.getMissingPartOfRecipe(recipe).ingredients.map(ingredient => {
-      let specificIngredient = ingredientData.find(item => item.id === ingredient.id);
-      return specificIngredient.name;
-    });
-
+    let missingIngredients = pantry.getMissingPartOfRecipe(recipe).ingredients.reduce((acc, ingredient) => {
+      if(ingredient.quantity.amount > 0) {
```

Comment:
check to make sure we don't have all that we need of the ingredient (we could exclude these ingredients earlier within the `getMissingPartOfRecipe()` function, but we'd have to change our `.map` to something else)